rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/demo_file.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\TypedPropertyRector\Fixture;
+
+use PHPUnit\Framework\MockObject\MockObject;
+
+class DemoFile
+{
+  /** @var Foo|mixed|MockObject */
+  private $x;
+}
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\TypedPropertyRector\Fixture;
+
+use PHPUnit\Framework\MockObject\MockObject;
+
+class DemoFile
+{
+  /** @var \Foo|\PHPUnit\Framework\MockObject\MockObject */
+  private mixed $x;
```

Comment:
mixed is not exists yet in php 7.4, so better skip it, ref https://3v4l.org/v1Vcd#v7.4.27

Reply:
Ok. So we do want to skip. I'll amend the PR.